mongodbatlas/resource_mongodbatlas_cluster.go

```diff
@@ -1081,10 +1078,6 @@ func resourceMongoDBAtlasClusterImportState(ctx context.Context, d *schema.Resou
 		return nil, fmt.Errorf("couldn't import cluster backup configuration %s in project %s, error: %s", *name, *projectID, err)
 	}
 
-	if err := d.Set("retain_backups_enabled", false); err != nil {
-		return nil, fmt.Errorf("couldn't import cluster backup configuration %s in project %s, error: %s", *name, *projectID, err)
-	}
-
```

Comment:
we should not set `retain_backups_enabled` as part of the import but only during cluster creation as the GET CLUSTER endpoint does not return `retain_backups_enabled`